packages/flutter/test/material/slider_test.dart

```diff
@@ -1210,22 +1211,35 @@ void main() {
       ),
     ));
 
+    await tester.pumpAndSettle();
+
+    debugDumpSemanticsTree(DebugSemanticsDumpOrder.traversalOrder);
```

Comment:
You're dumping the semantics tree at the wrong place. The test fails in line 1258. You should dump the semantics just before that and from that it seems like line 1258 needs to be changed to something like this:

```
    expect(
        semantics,
        hasSemantics(
          TestSemantics.root(
            children: <TestSemantics>[
              TestSemantics(
                id: 3,
              ),
            ],
          ),
          ignoreRect: true,
          ignoreTransform: true,
        ));
```

Reply:
Thanks, did not realize there were 2 expects here